Library/ViewModels/SettingsAccountViewModel.swift

```diff
@@ -44,13 +45,26 @@ SettingsAccountViewModelOutputs, SettingsAccountViewModelType {
     let shouldHideEmailWarning = userAccountFields.values()
       .map { response -> Bool in
         guard let isEmailVerified = response.me.isEmailVerified,
-              let isDeliverable = response.me.isDeliverable else {
-          return true
+          let isDeliverable = response.me.isDeliverable else {
+            return true
         }
 
         return isEmailVerified && isDeliverable
     }
 
+    let initialHiddenState = self.viewWillAppearProperty.signal.mapConst(true)
```

Comment:
Here I would also say we should default to assuming that users will be able to view that section, and hide it if not. @dannyalright thoughts on this?